src/components/Tabs.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+const Tabs = function({ tabs, selectTab, selectedTabId }) {
+  return (
+    <>
+      {tabs.map(tab => (
+        <button
+          type="button"
+          key={tab.id}
+          onClick={() => selectTab(tab)}
+        >
+          {tab.title}
+        </button>
+      ))}
+
+      {tabs.map((tab) => {
+        if (selectedTabId === tab.id) {
+          return (
+            <p key={tab.id}>{tab.content}</p>
+          );
+        }
+
+        return false;
+      })}
```

Comment:
Better to use find method to find active tab and display its content here.
The less JS logic is inside `jsx` - the better.